src/models/calendar-restful-model-collection.ts

```diff
@@ -159,4 +139,48 @@ export default class CalendarRestfulModelCollection extends RestfulModelCollecti
         return Promise.reject(err);
       });
   }
+
+  private buildAvailabilityPayload(
+    options: SingleAvailabilityQuery | ConsecutiveAvailabilityQuery
+  ): Record<string, unknown> {
+    // Instantiate objects from properties to get JSON formatted for the API call
+    const freeBusyJson = options.freeBusy
+      ? options.freeBusy.map(fb => new FreeBusy(fb).toJSON(true))
+      : [];
+    const openHoursJson = options.openHours
+      ? options.openHours.map(oh => new OpenHours(oh).toJSON(true))
+      : [];
+    const calendarsJson = options.calendars
+      ? options.calendars.map(cal => new FreeBusyCalendar(cal).toJSON(true))
+      : [];
+
+    return {
+      emails: options.emails,
+      duration_minutes: options.duration,
+      interval_minutes: options.interval,
```

Comment:
Is there a reason why `minutes` is omitted? It would be better to stay consistent with the API

Reply:
@james10424 that's a great callout, I'm not sure how it got missed during the refactor. Unfortunately if I were to make this change now, it would likely be considered a breaking change as people would need to adapt their code to the new keywords. Thus I wouldn't be able to make this change until the next major release. I've created a shortcut story to track this change when the time comes for the next major release though: https://app.shortcut.com/nylas/story/79594/update-availability-queries-to-match-api-notati